sharktank/sharktank/layers/testing.py

```diff
@@ -51,80 +51,139 @@ def make_llama_attention_block_theta(
     )
 
 
-def make_mmdit_double_block_theta(dtype: torch.dtype | None = None) -> Theta:
+def make_mmdit_double_block_theta(
```

Comment:
The naming would benefit if it clarified that it creates a random theta.

Reply:
Done.